.env

```diff
@@ -1 +1,2 @@
 SASS_PATH="node_modules"
+REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN=96db4b276ac372898c5fd4f8963133e360efa13a
```

Comment:
Your PAT should be in an `.env.local` file and not committed to your repo. Doing that leads to insecurities.

Reply:
If you look, I added to both, .env.local and .env. I just failed to remove from .env....Note: This token has already been removed. 

Please approve.

Reply:
`.env.local` does not get committed so I have no way of knowing what is in that file which is actually the point of putting your PAT in `.env.local` and not `.env` because then it does not get exposed to the world.